content/S2015/ S15-event-gooutside.md

```diff
@@ -0,0 +1,18 @@
+title: WiCS and CSC Go Outside
+Date: 2015-06-26 19:00
+Category: Events
+Tags: social
+Slug: go-outside-s15 
+Author: Fatema Boxwala 
+Summary: We're having a bonfire with the CSC there will be s'mores and froyo!
+
+Come out to the Velocity Firepit and enjoy the great outdoors with us and the CSC! There will be s'mores 
```

Comment:
It's now the "Velocity firepit"?? I'm assuming this is the EV3 firepit, beside the environment buildings and across the creek from Grebel. Didn't realize it was rebranded >.<

For old people like me you might want to put "(EV3 Firepit)" as a parenthetical.